pacman/model/graphs/application/application_vertex.py

```diff
@@ -184,13 +184,21 @@ def vertex_slices(self):
     def get_max_atoms_per_core(self):
         """ Gets the maximum number of atoms per core, which is either the\
             number of atoms required across the whole application vertex,\
-            or a lower value if a constraint lowers it.
+            or a lower value set.
 
         :rtype: int
         """
-        for constraint in self.constraints:
-            if isinstance(constraint, MaxVertexAtomsConstraint):
-                return constraint.size
+        return self._max_atoms_per_core
+
+    def set_max_atoms_per_core(self, new_value):
+        """
+        Set the max_atoms_per_core.
+
+        Can be used to raise the number of atoms.
+
+        :param int new_value: Value to set if not higher than previous
```

Comment:
Wrong description - it can be higher or lower

Reply:
agreed fixed